doc.go

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
What is this file's purpose?

Reply:
We intend to use it for "introductory" documentation, see https://blog.golang.org/godoc-documenting-go-code.
CC @joshgav

Reply:
Cool :D I didn't know about the doc.go feature